pxr/usd/bin/usdedit/usdedit.py

```diff
@@ -67,10 +67,9 @@ def _generateTemporaryFile(usdcatCmd, usdFileName, readOnly, prefix):
     (usdaFile, usdaFileName) = tempfile.mkstemp(
         prefix=fullPrefix, suffix='.usda', dir=os.getcwd())
 
-    # No need for an open file descriptor, as it locks the file in Windows.
+    subprocess.run([usdcatCmd, usdFileName], stdout=usdaFile)
```

Comment:
Should `usdFileName` be `usdaFileName`?

Reply:
No. `usdFileName` is the source file, `usdaFileName` is the temporary file created that we write to

(The names aren't great/descriptive for sure though)